test/Strings/compare.js

```diff
@@ -9,10 +9,12 @@ var str1 = "abcd1234"
 var str2 = "1234567a"
 var str3 = "abcd12345"
 var str1a = "abcd1234"
+var str1null = "\0ab\0cd1234\0";
 
 assert.isTrue(str1.localeCompare(str1a) == 0);
+assert.isTrue(str1.localeCompare(str1null) == 0);

```

Comment:
Seems like this is failing on non-ICU versions of chakracore, it seems that winglob doesn't ignore nulls in comparisons?

Reply:
Correct. I'm currently changing the the function to take the length of the strings.